README.md

```diff
@@ -132,6 +132,36 @@ required configuration is captured in the `Makefile`.
 
 ### Locally
 
+Create a .bundle directory in the root of the application
```

Comment:
@DanielCouzens in the future check to see if there's an `auth` make target first, this should do this for you. Let me know if this doesn't make sense?